code/modules/roguetown/roguejobs/fisher/leeches.dm

```diff
@@ -171,7 +171,7 @@
 		if(!get_location_accessible(H, check_zone(user.zone_selected)))
 			to_chat(user, span_warning("Something in the way."))
 			return
-		var/used_time = (70 - (H.mind.get_skill_level(/datum/skill/misc/treatment) * 10))/2
+		var/used_time = (70 - (user.mind.get_skill_level(/datum/skill/misc/treatment) * 10))/2
```

Comment:
```suggestion
		var/used_time = 3.5 SECONDS - user.mind.get_skill_level(/datum/skill/misc/treatment)/2 SECONDS
```
While you're at it, it'd be nice if you made it clearer what the calculation is. You could also do `(3.5 - user.mind.get_skill_level(/datum/skill/misc/treatment)/2) SECONDS` instead if you think that's clearer.

Reply:
...This winds up being a differant calculation then the one they have made, I think.

Reply:
It shouldn't be. `(70 - (treatment skill * 10))/2` is equivalent.
1. Through the distributive property, you get `70/2 - treatment_skill*10/2`.
2. Through substitution, we replace `*10` with `SECONDS`. `70/2 - treatment_skill SECONDS / 2`
3. Through the commutative property we reorder the second term. `70/2 - treatment_skill/2 SECONDS` (note that this is `treatment_skill / 2 * 10`, not `treatment_skill / (2 * 10)`.)
4. Through arithmetic we get `35 - treatment_skill / 2 SECONDS`.
5. Representing 35 in terms of seconds, we can use `3.5 SECONDS` instead of `35`.
So, if you want to make it clearer you can use parentheses for `(treatment_skill / 2) SECONDS`, but by order-of-operations rules they aren't strictly necessary.